packages/cli/src/index.test.ts

```diff
@@ -70,147 +64,6 @@ describe('CLI', () => {
     expect(processError).toBeCalledWith(expect.stringContaining(`error: unknown command 'xyz'`));
   });
 
-  test('Client ID and secret login', async () => {
-    process.env.MEDPLUM_CLIENT_ID = '123';
-    process.env.MEDPLUM_CLIENT_SECRET = 'abc';
-    await main(medplum, ['node', 'index.js', 'whoami']);
-    expect(medplum.getAccessToken()).toBeDefined();
-  });
-
-  test('Login success', async () => {
-    (cp.exec as unknown as jest.Mock).mockReturnValue(true);
-    (http.createServer as unknown as jest.Mock).mockReturnValue({
-      listen: () => ({
-        close: () => undefined,
-      }),
-    });
-
-    // Expect no active login to start
-    expect(medplum.getActiveLogin()).toBeUndefined();
-
-    // Start the login
-    await main(medplum, ['node', 'index.js', 'login']);
-
-    // Get the handler
-    const handler = (http.createServer as unknown as jest.Mock).mock.calls[0][0];
-
-    // Simulate a favicon.ico request, don't crash
-    const req1 = { url: '/favicon.ico' };
-    const res1 = { writeHead: jest.fn(), end: jest.fn() };
-    await handler(req1, res1);
-    expect(res1.writeHead).toBeCalledWith(404, { 'Content-Type': 'text/plain' });
-    expect(res1.end).toBeCalledWith('Not found');
-
-    // Simulate the redirect
-    const req2 = { url: '/?code=123' };
-    const res2 = { writeHead: jest.fn(), end: jest.fn() };
-    await handler(req2, res2);
-    expect(res2.writeHead).toBeCalledWith(200, { 'Content-Type': 'text/plain' });
-    expect(res2.end).toBeCalledWith('Signed in as Alice Smith. You may close this window.');
-    expect(medplum.getActiveLogin()).toBeDefined();
-  });
-
-  test('Load credentials from disk', async () => {
-    medplum = new MockClient({ storage: new FileSystemStorage() });
-
-    (fs.existsSync as unknown as jest.Mock).mockReturnValue(true);
-    (fs.readFileSync as unknown as jest.Mock).mockReturnValue(
-      JSON.stringify({
-        activeLogin: JSON.stringify({
-          accessToken: 'abc',
-          refreshToken: 'xyz',
-          profile: {
-            reference: 'Practitioner/123',
-            display: 'Alice Smith',
-          },
-          project: {
-            reference: 'Project/456',
-            display: 'My Project',
-          },
-        }),
-      })
-    );
-
-    await main(medplum, ['node', 'index.js', 'whoami']);
-
-    expect((console.log as unknown as jest.Mock).mock.calls).toEqual([
-      ['Server:  https://example.com/'],
-      ['Profile: Alice Smith (Practitioner/123)'],
-      ['Project: My Project (Project/456)'],
-    ]);
-  });
```

Comment:
move to https://github.com/medplum/medplum/pull/2206/files#diff-32f0901bf847ac4f6b89540d53766e03661dce5f12d13a51d543452c15952643R85-R104